libpod/container_inspect.go

```diff
@@ -127,6 +127,17 @@ func (c *Container) getContainerInspectData(size bool, driverData *inspect.Data)
 		IsInfra: c.IsInfra(),
 	}
 
+	if c.config.HealthCheckConfig != nil {
+		//	This container has a healthcheck defined in it; we need to add it's state
+		healthCheckState, err := c.ReadHealthCheckLogFromFile()
```

Comment:
Do we need code to add the healthcheck command to inspect too? I see it in the inspect struct but not here

Reply:
it is there

Reply:
SGTM then